demo/addons/fmod/fmod_config_window.gd

```diff
@@ -0,0 +1,100 @@
+tool
+extends VBoxContainer
+class_name FMOD_Config
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	get_node("%ApplyChanges").connect("pressed", self, "apply_changes")
+	get_node("%UndoChanges").connect("pressed", self, "load_file")
+	get_node("%LoadBanksButton").connect("pressed", self, "load_bank_names")
+	get_node("%ClearBanksButton").connect("pressed", self, "clear_bank_names")
+	get_node("%DoplerScaleSlider").connect("value_changed", self, "update_dop_val_label")
+	get_node("%RolloffValue").connect("value_changed", self, "update_rolloff_scale_label")
+	
+	load_file()
+
+func load_file():
+	var config = ConfigFile.new()
+	
+	var err = config.load("res://addons/fmod/fmod_config.cfg")
+
+	if err != OK:
+		#Set Default Values
+		config.set_value("Init", "SRIndex", 1)
+		config.set_value("Init", "SampleRate", 44100)
+		config.set_value("Init", "LiveUpdate", false)
+		config.set_value("Init", "SpeakerMode", 3)
+		config.set_value("Banks", "Location", "Hello There Friend")
+		config.set_value("Init", "NumChannels", 64)
+		return
+
+	#Change input locations to show loaded values
+	get_node("%SampleRateSelector").selected = config.get_value("Init", "SRIndex")
```

Comment:
Can we move all these into `onready` variables? As otherwise Godot will attempt to do a lookup on the scene tree for each of these every time `load_file()` is invoked.